config/locales/client.en.yml

```diff
@@ -10,7 +10,8 @@ en:
   site_settings:
     zendesk_url: Your Zendesk URL. e.g. https://YOUR_SUBDOMAIN.zendesk.com/api/v2
     zendesk_enabled: Enable Zendesk plugin
-    zendesk_enabled_categories: List of categories that will autogenerate Zendesk tickets
+    zendesk_all_categories: Create tickets for all categories? If enabled, `zendesk_enabled_categories` are ignored.
```

Comment:
I recommend changing the setting name to `zendesk_enable_all_categories`.

Also, let's replace the question mark `?` with period `.` in setting description for the sake of consistency.